src/ui/base-page-storyEdit.js

```diff
@@ -77,17 +132,22 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             "video": ["mediaUrl"]
         },
         components: {
+            // manaages browser history for in-page forward-back support
+            historian: {
+                type: "gpii.locationBar",
+                options: {
+                    model: {
+                        editorVisible: "{storyEdit}.model.editorVisible",
```

Comment:
Add a code comment to explain that only these 2 model values are tracked because transformations between:
1. `{storyEdit}.model.editorVisible` and `{storyEdit}.model.previewerVisible`
2. `{storyEditor}.model.editStoryStepVisible` and `storyEditor}.model.metadataStepVisible`
will set other model values properly.

Reply:
bump

Reply:
I misread your suggestion, have added a comment here :)